src/extensions/mvs/load.ts

```diff
@@ -216,6 +224,29 @@ const MolstarLoadingActions: LoadingActions<MolstarTree, MolstarLoadingContext>
             colorTheme: colorThemeForNode(node, context),
         });
     },
+    volume_representation(updateParent: UpdateTarget, node: MolstarNode<'volume_representation'>, context: MolstarLoadingContext): UpdateTarget {
+        const visualParams = {
+            'type': {
+                'name': node.params.type,
+                'params': {}
+            },
+            'colorTheme': colorThemeForNode(node, context),
+            'sizeTheme': {
+                'name': 'uniform',
+                'params': {
+                    'value': 1
+                }
+            }
+        };
+        const params = {
+            ...visualParams,
+            ...volumeRepresentationProps(node.params),
+        };
+        const update = UpdateTarget.apply(updateParent, VolumeRepresentation3D, {
+            ...params
```

Comment:
No need to use `{...params}` instead of just `params` here. Or am I missing something.

Reply:
Yes, you are right, I have overdid it a bit 😄. Removed that.